src/components/ScenarioParameters/components/BasicTypes.js

```diff
@@ -89,6 +97,12 @@ const BasicTypes = ({
           changeSwitchType={changeSwitchType}
           switchProps={switchFieldProps}
         />
+        <BasicDateInput
+          classes={classes}
+          label='Pick a date'
```

Comment:
Could you please use 'useTranslation' hook here and above to change label regarding language?